infra/notifications/templates/messageTemplates.js

```diff
@@ -127,6 +133,15 @@ const messageTemplates = {
         text: template(
           fs.readFileSync(`${templateDir}/seller-OfferFinalized.txt`).toString()
         )
+      },
+      OfferData: {
```

Comment:
We have OfferFinalized, OfferCreated, ...Offer[Action]. It would be nice to name this accordingly `OfferReviewed` or something to that nature.

I would like an Origin team member to weigh in on this before you make any changes.

Reply:
`OfferData` is event name from our Solidity contracts. Don't think we plan on upgrading the contracts anytime soon. So, I'd suggest to leave it as it is for now.